### ISSUE TITLE
[BUG] Desecration eats Earthen Power ticks

### ISSUE THREAD

**Issue by freddaxyz** [JamminL/wotlk-classic-bugs#695]
#### 1.1. Description
since 3.4.2, Earthen power attempts (unsuccessfully) to dispel Desecration slow even though Desecration is a continuous on-the-ground area effect slow which Earthen Power should ignore.
  #### 1.2. How to Reproduce
``` 
1. Have a death knight with 2/2 Desecration use Plague Strike or Scourge Strike on a shaman with 2/2 Earthen Power to put Desecration on the ground underneath them
2. Have the death knight use Chains of Ice on the shaman
3. Have the shaman use Earthbind Totem with 2/2 Earthen power while in Desecration slow and slowed by Chains of Ice
4. Repeat steps 1-3 (alternatively leave Earthbind to tick multiple times) and observe Chains of Ice only being dispelled 50% of the time over a large sample size instead of 100% prior to 3.4.2
```
  #### 1.3  Source Material
https://streamable.com/l1smkz - demonstrating how in 3.4.2 Earthen Power attempts to dispel Desecration but the slow itself never fades even for a split second (combat text addon helps identify what was dispelled as seen to the right of player character, displayed as "Pet Dispel! [Debuff name]")

https://streamable.com/u446cz - demonstrating how in 3.4.2 Earthen Power dispels Desecration repeatedly while Chains of Ice slow is present

#### 2.1. Description
Earthen Power should never attempt to remove an area of effect that continuously applies its slow, similar to how doing the same thing with a hunter's Frost Trap and another slow will never attempt to dispel the Frost Trap, which is how Desecration also (correctly) worked prior to patch 3.4.2.

alternatively, a compromise could be Earthen Power actually dispelling the Desecration slow (until re-entering the AOE, or a grace period of X.Y seconds where you don't get re-slowed) but soaking the slow removal while not having a slow removed seems not so fair.
  #### 2.2 Source Material
https://streamable.com/ck6uhp - footage showing how Earthen Power never attempts to dispel Frost Trap, a similar spell 

https://streamable.com/bu1zdv - footage from season 6 arena showing how with Earthbind Totem up and ticking for the entirety of the 20 second clip, Earthen Power never attempts to dispel Desecration, and once other slows are applied it goes straight to them instead of Desecration (no dispel attempts until Chains of Ice ~10 seconds into the clip, Crippling Poison at ~13 seconds, and Chains of Ice again at ~16 seconds)

**Comment by raethkcj** [comment]
Should be fixed as of 3.4.3, I believe the actual change is that Earthen Power removes up to 99 slows per tick

https://us.forums.blizzard.com/en/wow/t/wrath-of-the-lich-king-classic-patch-notes-version-343/1685291

**Comment by JamminL** [comment]
Correct, fixed in this new patch
